internal/data-ingest-cli/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"github.com/Dynatrace/dynatrace-otel-collector/internal/data-ingest-cli/commands/otlpjson"
+	"log"
+)
+
+func main() {
+	// Define the CLI arguments
+	inputFile := flag.String("input-file", "", "Path to the input file containing input data")
+	collectorURL := flag.String("collector-url", "", "URL of the OpenTelemetry collector")
+	outputFile := flag.String("output-file", "", "Path to the file where received OTLP data will be stored")
+	inputFormat := flag.String("input-format", "otlp-json", "Input format (options: 'otlp-json', 'syslog', 'systemd')")
+	otlpSignalType := flag.String("otlp-signal-type", "", "OTLP signal type (options: 'logs', 'traces', 'metrics')")
+	receiverPort := flag.Int("receiver-port", 0, "OTLP Receiver port. If set, the tool will open a grpc server on the specified port to receive data and store it in an output file")
+
+	// Parse the CLI arguments
+	flag.Parse()
+
+	// Validate required arguments
+	if *collectorURL == "" {
```

Comment:
I know this is contrary to having a default for the collector URL, but should this fail if there is no collector URL? I can see this tool being deployed somewhere simply to extract some test data for reproducible problem cases, and you might not want to send data out in that case

Reply:
yes good point - i will check and see if i can refine the parameter validation a bit more and set some defaults where applicable

Reply:
actually, right now the tool is not really designed to run as only a OTLP receiver, as it will terminate as soon as it has received the precessed data item it has sent previously.
However, it might be useful to also support the use case you mentioned, since the dt-collector does not include the file exporter. We could think of adding this in a follow up PR, wdyt?